### PR TITLE
proposed charter revision

### PR DESCRIPTION
**Pull request by Benjmhart**

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -19,135 +16,76 @@ The Development Experience Working Group Charter establishes the Scope, intellec
...
-https://github.com/input-output-hk/Developer-Experience-working-group
-
-
-   7. Roles
-Repository Maintainer 
-Maintainers are expected to handle the repository management as described in the Repository Process.
-Repository Maintainer selection
-Maintainers must be Working Group Participants selected by Working Group Participants via consensus. At least 3 maintainers to be active at any given time operating the Repository Process for the Working Group. Maintainers must have different company affiliations to represent the diversity of the Working Group. 
-
+Issues/Proposals can have the following statuses
```
Comment (codybutz) [inline CHARTER.md:84]: Is the intention here that a developer would tag their issue with a label to mark it under one of these statuses? Do we need to create an issue template for GitHub issues?

File: CHARTER.md [hunk 2]
```diff
@@ -194,22 +114,3 @@ Feedback agreement
...
-
-
-
-
-
-
-
-
-
-[a]Rotating leadership is great - The details are a bit premature I think... I don't know that every 2 months is possible. Perhaps, once a year?
```
Comment (codybutz) [inline CHARTER.md:215]: Did anything ever come of this? Is the plan to have rotating leadership?

**Comment by AdsIO** [comment]
3. Chair Selection
Chairpersons:  
* Ben Hart
* Manu Gunther


I will propose members from the community to be chairs, in this case Manu wont be appropriate as he represents IO. 
@Benjmhart @codybutz @manu what are your thoughts?

File: CHARTER.md (hunk 2 above)
Comment (Benjmhart) [reply CHARTER.md:215]: 100%   4 concurrent chairpeople, ideally dividing responsibilities either round-robin or based on preference,  and terms of 6 months?


How do terms work for CIP editors?

File: CHARTER.md (hunk 1 above)
Comment (Benjmhart) [reply CHARTER.md:84]: eventually we will indeed!

also a proposal template, a draft of which is already in PR.

File: CHARTER.md [hunk 3]
```diff
@@ -19,135 +16,77 @@ The Development Experience Working Group Charter establishes the Scope, intellec
...
-
+TODO: provide links, start non-IOG discord?
 
 This is a community-led initiative so we encourage members of the Cardano community who want to improve the overall developer experience across the cardano ecosystem to join the Developer Experience Working Group start by joining the Discord Channel 
 
 
 1. Join our discord server (use this link), and verify.
+https://discord.gg/inputoutput 
 2. Submit your discord handle in this single-field form
+????
```
[the remark this answers is not in the document]
Comment (xem2022) [reply CHARTER.md:70]: @Benjmhart - see below: 
Join the discord server https://discord.gg/WmSVtQ5PjZ
Submit your discord handle https://input-output.typeform.com/to/jKGyuidt?typeform-source=mail.google.com

**Review by AdsIO — APPROVED** [review]
Approved as agreed at meeting held on the 18th October

**Pull request merged**